README.md

```diff
@@ -213,7 +213,7 @@ const columns = [
 |**`empty`**|boolean|false|This denotes whether the column has data or not (for use with intentionally empty columns)
 |**`viewColumns`**|boolean|true|Allow user to toggle column visibility through 'View Column' list
 |**`filterList`**|array||Filter value list [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js)
-|**`filterOptions`**|array||Filter options [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js)
+|**`filterOptions`**|{names, logic}||Filter options. it's possible use an array of names [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js) or an object with names and logic [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/customize-filter/index.js)
```

Comment:
How about instead of `it's possible use an array of names` we do `although it's still possible to use an array of names, this option has been deprecated`

Reply:
ok